shared/message/skill/Look.proto

```diff
@@ -23,8 +23,8 @@ import "Vector.proto";
 
 /// A message telling the robot to look in the given direction
 message Look {
-    /// Vector from the camera to the point to look at, in camera space
-    vec3 rPCc = 1;
+    /// Vector from the torso to the point to look at, in torso space
+    vec3 rPCt = 1;
```

Comment:
Comment should read "Vector from the camera to the point to look at, in torso space"

Reply:
or rPTt if comment is correct